files/opt/upgrade.sh

```diff
@@ -10,25 +10,25 @@ if [[ $REPLY =~ ^[Yy]$|^$ ]]; then
     yarn set version classic
 
     echo "Downloading new Mastodon code..."
-    su - mastodon -c "cd /home/mastodon/live && git fetch --tags && git checkout $(git tag -l | grep '^v[0-9.]*$' | sort -V | tail -n 1)"
+    su - mastodon -c "cd /home/mastodon/live && git fetch --tags && git checkout $(su - mastodon -c 'cd /home/mastodon/live && git tag -l | grep '^v[0-9.]*$' | sort -V | tail -n 1')"
```

Comment:
I dont really like this, there must be a better way.

Maybe using an intermediate variable to contain the result of the subcommand?

Reply:
Hmm possibly...I will experiment with this a bit.

Reply:
Alright, this is a lot better now.